js/App.js

```diff
@@ -125,11 +126,43 @@ App.prototype.setUnreadChatMessageCount = function(count) {
   }
 };
 
-App.prototype.intlNumFormat = function(numberToFormat, maxDigits){
-  maxDigits = maxDigits || 8; //default to show down to the satoshi (.00000001)
+App.prototype.intlNumFormat = function(numberToFormat, maxDigits = 8) {
   return new Intl.NumberFormat(window.lang, {maximumFractionDigits: maxDigits}).format(numberToFormat);
 };
 
+App.prototype.getBitcoinUnit = function() {
+  if (!this.bitcoinUnit) {
```

Comment:
Can we please preface the `bitcoinUnit' property with an underscore to indicate it's private? I wouldn't want someone to mistakingly modify that and not use the getter / setter functions. It's obviously not enforced, but at least by convention, the dev has an idea that the property is not meant to be publicly altered.

`if (!this.bitcoinUnit) {`    --->    `if (!this._bitcoinUnit) {`